tests/system/action/list_of_speakers/test_delete_all_speakers.py

```diff
@@ -29,6 +30,41 @@ def test_delete_all_correct(self) -> None:
     def test_wrong_id(self) -> None:
         response = self.request("list_of_speakers.delete_all_speakers", {"id": 111})
         self.assert_status_code(response, 400)
-        self.assertTrue(
+        assert (
             "Model 'list_of_speakers/111' does not exist." in response.json["message"]
         )
+
+    def test_delete_all_no_permissions(self) -> None:
+        self.base_permission_test(
+            {
+                "meeting/1": {
+                    "speaker_ids": [1],
+                },
+                "list_of_speakers/111": {
+                    "closed": False,
+                    "meeting_id": 1,
+                    "speaker_ids": [1],
+                },
+                "speaker/1": {"list_of_speakers_id": 111, "meeting_id": 1},
+            },
+            "list_of_speakers.delete_all_speakers",
+            {"id": 111},
+        )
+
+    def test_delete_all_permissions(self) -> None:
+        self.base_permission_test(
+            {
+                "meeting/1": {
+                    "speaker_ids": [1],
+                },
+                "list_of_speakers/111": {
+                    "closed": False,
+                    "meeting_id": 1,
+                    "speaker_ids": [1],
+                },
+                "speaker/1": {"list_of_speakers_id": 111, "meeting_id": 1},
+            },
```

Comment:
Same here: save it into a variable instead of writing it twice. Same probably applies for more positions from #616, which I've missed there...

Reply:
Done.